receiver/otlpreceiver/otlp.go

```diff
@@ -185,10 +191,57 @@ func (r *otlpReceiver) registerTraceConsumer(tc consumer.Traces) error {
 		r.httpMux.HandleFunc("/v1/traces", func(resp http.ResponseWriter, req *http.Request) {
 			handleUnmatchedRequests(resp, req)
 		})
+
+		r.httpMux.HandleFunc(
+			"/opentelemetry.proto.collector.trace.v1.TraceService/Export",
+			func(resp http.ResponseWriter, req *http.Request) {
+				prepareGrpcResponse(resp)
+				body, err := deframeGrpc(req)
+				if err != nil {
+					writeGrpcError(resp, err)
+					return
+				}
+				otlpReq, err := pbEncoder.unmarshalTracesRequest(body)
+				if err != nil {
+					writeGrpcError(resp, err)
+					return
+				}
+				otlpResp, err := r.traceReceiver.Export(req.Context(), otlpReq)
+				if err != nil {
+					writeGrpcError(resp, err)
+					return
+				}
+				msg, err := pbEncoder.marshalTracesResponse(otlpResp)
+				if err != nil {
+					writeGrpcError(resp, err)
+					return
+				}
+				writeGrpcMessage(resp, msg)
+			}).Methods(http.MethodPost)
 	}
 	return nil
 }
 
+func writeGrpcMessage(resp http.ResponseWriter, msg []byte) {
+	_, err := resp.Write([]byte{0})
```

Comment:
Wouldn't be hard to implement compressed responses, didn't bother as we know for now the responses are always empty